dtfs-central-api/src/helpers/amendments/tfm-facility-latest-completed-amendment-to-facility-value.test.ts

```diff
@@ -0,0 +1,101 @@
+import { TfmFacility, TfmFacilityAmendment } from '@ukef/dtfs2-common';
+import { getLatestCompletedAmendmentToFacilityValue } from './tfm-facility-latest-completed-amendment-to-facility-value';
+import { aTfmFacility, aTfmFacilityAmendment } from '../../../test-helpers';
+
+describe('tfm-facility-latest-completed-amendment-to-facility-value', () => {
+  describe('getLatestCompletedAmendmentFacilityValue', () => {
+    it('returns undefined when the supplied tfm facility amendments are undefined', () => {
+      // Arrange
+      const tfmFacility: TfmFacility = {
+        ...aTfmFacility(),
+        amendments: undefined,
+      };
+
+      // Act
+      const result = getLatestCompletedAmendmentToFacilityValue(tfmFacility);
+
+      // Assert
+      expect(result).toBeUndefined();
+    });
+
+    it('returns undefined when the supplied tfm facility amendments array is empty', () => {
+      // Arrange
+      const tfmFacility: TfmFacility = {
+        ...aTfmFacility(),
+        amendments: [],
+      };
+
+      // Act
+      const result = getLatestCompletedAmendmentToFacilityValue(tfmFacility);
+
+      // Assert
+      expect(result).toBeUndefined();
+    });
+
+    it('returns undefined when the supplied tfm facility has no completed amendments', () => {
+      // Arrange
+      const amendments: TfmFacilityAmendment[] = [
+        { ...aTfmFacilityAmendment(), status: 'In progress', value: 100000 },
```

Comment:
Could we use `AMENDMENT_STATUS.IN_PROGRESS` for this? Similar changes would be required to the other statuses below

(Similar to us trying to replace currencies or fee record statuses in new PRs.)

Reply:
Cool, thanks